rllib/evaluation/rollout_worker.py

```diff
@@ -1457,7 +1457,7 @@ def get_filters(self, flush_after: bool = False) -> Dict:
         return return_filters
 
     @DeveloperAPI
-    def save(self) -> bytes:
+    def get_state(self) -> bytes:
```

Comment:
Renamed for clarity:
`get_state()` -> always return dict (see Policy or Algorithm)
`save()` -> Saves to disk and returns path.